### DISCUSSION TITLE
Bulk upload works despite filterOptions when uploading directly to a field

### DISCUSSION THREAD

**Discussion by paulpopus** [payloadcms/payload#15702] [Feature Requests & Ideas]
### Environment Info

```text
latest beta
```

### Describe the Bug

Bulk upload works despite filterOptions when uploading directly to a field

### Reproduction Steps

/

**Comment by JarrodMFlesch** [comment]
Seems like mimeTypes should be a config option on upload fields and those should be merged into the filterOptions. This will allow the frontend to use mimeTypes and prevent users from uploading incorrect mimeType files. 
- Should add mimeTypes specific property
- Should add field validation for upload fields to ensure that the file being uploaded is valid on the field level
- Should add frontend logic so the file selector can scope valid file types when selecting
